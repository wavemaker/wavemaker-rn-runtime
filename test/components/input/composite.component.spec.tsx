import React, { ReactNode } from 'react';
import { Text, TouchableOpacity } from 'react-native';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import WmComposite from '@wavemaker/app-rn-runtime/components/input/composite/composite.component';

describe('Test Composite component', () => {
    test('Check validity of sample component', () => {
      const tree = renderer.create(<WmComposite name="test_Composite"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});