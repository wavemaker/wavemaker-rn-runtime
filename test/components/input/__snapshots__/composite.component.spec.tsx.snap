// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Composite component Check validity of sample component 1`] = `
<View
  style={
    {
      "alignItems": "flex-start",
      "flexDirection": "row",
    }
  }
/>
`;
